include/freebsd/osd.h

```diff
@@ -35,6 +35,7 @@
 
 #include <sys/endian.h>
 #include <pthread_np.h>
+#include <netinet/in.h>
```

Comment:
> The comment here (https://github.com/ofiwg/libfabric/blob/master/util/pingpong.c#L345) indicates any protocol, which should be 0, but then specifies IPPROTO_TCP. @bturrubiates is it necessary to do that?

I think the comment is just wrong. We want the out of band communication to be over TCP.